code/modules/species/species.dm

```diff
@@ -689,13 +689,14 @@ var/global/const/DEFAULT_SPECIES_HEALTH = 200
 	for(var/accessory in get_available_accessory_types(bodytype, accessory_category))
 		. += GET_DECL(accessory)
 
-/decl/species/proc/get_available_accessory_types(bodytype, accessory_category)
+/decl/species/proc/get_available_accessory_types(decl/bodytype/bodytype, accessory_category)
 	if(!bodytype)
 		bodytype = default_bodytype
-	var/list/available_accessories = LAZYACCESS(accessory_styles, accessory_category)
+	var/list/available_accessories = accessory_styles?[bodytype.type]?[accessory_category]
```

Comment:
howso? bodytype should never be null here, and since we're not indexing numerically, `?[x]` is equivalent to `LAZYACCESS` here. it's just an easier way to do chained lazyaccess